public/js/script.js

```diff
@@ -0,0 +1,73 @@
+// setInterval(gameLoop, 33); // 33 milliseconds = ~ 30 frames per sec
+//Creating drawing vars
+var canvas;
```

Comment:
Everything here looks good, if you would go back and change from the js es5 'var' usage to the modern es6 'let' so we can stay current with best practices.